### ISSUE TITLE
Bad Request: sticker set name invalid

### ISSUE THREAD

**Issue by Defuera** [tdlib/td#466]
Trying to call `createNewStickerSet`, but getting an error
```
error_code=400, description='Bad Request: sticker set name invalid'
```
params:
```
user_id = 1111,
name =  "what_den_things",
title =  "What Denis thinks",
png_sticker     "BQADAgADxQIAAsQQMUqjvj3PCWFVMwI", // I tried url, file, file_id here, seems it has nothing to do with the problem
emoji: = "👍"
```
I tried also `uploadStickerFile` method, works like a charm.
What am I doing wrong?

**Comment by levlam** [comment]
Those methods works for bots only and has some restriction, as described in https://core.telegram.org/bots/api#createnewstickerset. In particular, sticker set name should end with `_by_<bot username>`.

**Comment by Defuera** [comment]
Hey man, thank you! I'll check it later if it works. I might have missed it, by I think documentation was showing  `by<bot username>` before. And I tried it and it didn't work. 
Have you updated it?

**Comment by levlam** [comment]
There was a problem with documentation displaying yesterday for a few hours.

**Comment by levlam** [comment]
@beastzx18 Are you sure that the sticker set was created by the @beastzvbot and not by the user itself through @stickers bot?

**Comment by beastzx18** [comment]
yes i am sure i can give the object json form if it is needed for proof.


{
    "_": "types.messages.StickerSet",
    "set": {
        "_": "types.StickerSet",
        "id": 6111668973932118033,
        "access_hash": 4448188884844802286,
        "title": "mystickerset",
        "short_name": "tron12345678sticker_by_beastzvbot",
        "count": 1,
        "hash": -869546480,
        "archived": false,
        "official": false,
        "masks": false,
        "animated": false,
        "videos": false,
        "installed_date": 1657301833,
        "thumbs": []
    },
    "packs": [
        {
            "_": "types.StickerPack",
            "emoticon": "👍",
            "documents": [
                6217458021342120027
            ]
        }
    ],
    "documents": [
        {
            "_": "types.Document",
            "id": 6217458021342120027,
            "access_hash": -4316565899611370592,
            "file_reference": "b'\\x00b\\xc8kI\\x8d\\rj\\xb6\\xfd\\x8a\\xe1\\xd9\\xcb\\x8c\\xa0\\xf1\\x9b\\xa7\\x0f\\xe6'",
            "date": 1657301833,
            "mime_type": "image/webp",
            "size": 56824,
            "dc_id": 5,
            "attributes": [
                {
                    "_": "types.DocumentAttributeImageSize",
                    "w": 512,
                    "h": 512
                },
                {
                    "_": "types.DocumentAttributeSticker",
                    "alt": "👍",
                    "stickerset": {
                        "_": "types.InputStickerSetID",
                        "id": 6111668973932118033,
                        "access_hash": 4448188884844802286
                    },
                    "mask": false
                },
                {
                    "_": "types.DocumentAttributeFilename",
                    "file_name": "sticker.webp"
                }
            ],
            "thumbs": [
                {
                    "_": "types.PhotoPathSize",
                    "type": "j",
                    "bytes": "b'\\x19\\x06\\xb3\\x02\\xe1Y\\x06\\xe5\\x00Y\\x06\\x80Y\\x06\\xe1\\x19\\x06\\x99\\x06\\xef\\x19\\x06\\x99\\x06\\xe1Y\\x06'"
                },
                {
                    "_": "types.PhotoSize",
                    "type": "m",
                    "w": 320,
                    "h": 320,
                    "size": 24028
                }
            ],
            "video_thumbs": []
        }
    ]
}

**Comment by levlam** [comment]
@beastzx18 This object is definitely not from Bot API. Did you create the sticker set from the user's account using stickers.createStickerSet?

**Comment by beastzx18** [comment]
sorry i didn't tell you, but i am using pyrogram and pyrogram raw methods, but the rules to add a stickers in a sticker set should be same.

and i created the sticker set from bot client of pyrogram

**Comment by levlam** [comment]
The sticker set wasn't created by the bot, and hence it can't edit it. This should be the only way to receive the "STICKERSET_INVALID" error.

**Comment by beastzx18** [comment]
okay thanks for the help !
